pyanaconda/installclasses/installclass_atomic.py

```diff
@@ -0,0 +1,143 @@
+#
+# installclass_atomic.py
+#
+# Atomic-specific partitioning defaults
+#
+# Copyright (C) 2014  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import os
+import shutil
+from pyanaconda.installclasses.fedora import FedoraBaseInstallClass
+from pyanaconda.constants import *
+from pyanaconda.product import *
+from pyanaconda import network
+from pyanaconda import nm
+from pyanaconda.core import util
+import types
+from pyanaconda.kickstart import getAvailableDiskSpace
+from pyanaconda.storage.partspec import PartSpec
+from pyanaconda.storage.autopart import swap_suggestion
+from pyanaconda.platform import platform
+from blivet.size import Size
+
+import logging
+log = logging.getLogger("anaconda")
+
+class AtomicInstallClass(FedoraBaseInstallClass):
+    name = "Atomic Host"
+    sortPriority = 11000
+    hidden = False
+    defaultFS = "xfs"
+
+    def __init__(self):
+        self.localemap = {} # loaded lazily
+        FedoraBaseInstallClass.__init__(self)
+
+    # This is intended right now to match Fedora Server; if changing this,
+    # please discuss on https://lists.projectatomic.io/projectatomic-archives/atomic-devel/
```

Comment:
Side note...now that this is in the same codebase as Server, maybe we could actually dedup this?  Not a blocker/requirement for merging though from my PoV.

Reply:
I've used default partitioning from Fedora Server in the Fedora Atomic install class. Is that what you meant?